testsuite/policy/dns_policy.py

```diff
@@ -57,6 +59,7 @@ def set_health_check(self, health_check: HealthCheck):
         """Sets health check for DNSPolicy"""
         self.model["spec"]["healthCheck"] = asdict(health_check)
 
+    @backoff.on_exception(backoff.fibo, OpenShiftPythonException, max_tries=13, jitter=None)
```

Comment:
How time consuming is oc.selector() call?

potential time spent here is 13*call_elapsed + 144 [sec], which might be quite a lot.